librz/core/cmd_debug.c

```diff
@@ -4219,6 +4216,39 @@ RZ_IPI int rz_debug_continue_oldhandler(void *data, const char *input) {
 	return 1;
 }
 
+RZ_IPI RzCmdStatus rz_cmd_debug_step_until_handler(RzCore *core, int argc, const char **argv) {
+	rz_reg_arena_swap(core->dbg->reg, true);
+	step_until(core, rz_num_math(core->num, argv[1]));
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_cmd_debug_step_until_instr_handler(RzCore *core, int argc, const char **argv) {
+	step_until_inst(core, argv[1], false);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_cmd_debug_step_until_instr_regex_handler(RzCore *core, int argc, const char **argv)  {
+	step_until_inst(core, argv[1], true);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_cmd_debug_step_until_optype_handler(RzCore *core, int argc, const char **argv) {
+	char *optypes = strdup(rz_str_trim_head_ro((char *)argv[0]));
```

Comment:
As said, this does not work. Please have a look at how `RZ_CMD_ARG_FLAG_ARRAY` is used in other handlers. For example, I suggest looking at cmd_zign.yaml.